_dev/cldr/number-formatter.js

```diff
@@ -251,15 +252,20 @@ class CurrencyFormatter {
    * @return mixed
    */
   performSpecificReplacements(formattedNumber) {
-    return formattedNumber
-      .split(CURRENCY_SYMBOL_PLACEHOLDER)
-      .join(this.numberSpecification.getCurrencySymbol());
+    if (this.numberSpecification instanceof PriceSpecification) {
+      return formattedNumber
+        .split(CURRENCY_SYMBOL_PLACEHOLDER)
+        .join(this.numberSpecification.getCurrencySymbol());
+    }
+
+    return formattedNumber;
   }
 
   static build(specifications) {
     const symbol = new NumberSymbol(...specifications.symbol);
```

Comment:
You can have a symbol even when it's not a price formatting?

Reply:
Yep it's not a currency symbol, it's a list of symbol for the number format  https://github.com/PrestaShop/PrestaShop/blob/develop/src/Core/Localization/Specification/NumberSymbolList.php